gammapy/utils/table.py

```diff
@@ -13,6 +15,11 @@
 ]
 
 
+class GammapyMaskedConstant(MaskedConstant):
```

Comment:
Add a docstring to explain why this is needed.

Reply:
removed